modules/steam.nix

```
@@ -439,5 +471,79 @@ in
         WINE_CPU_TOPOLOGY = "8:0,1,2,3,4,5,6,7";
       };
     })
+    (mkIf cfg.autoStart {
+      assertions = [
+        {
+          assertion = !config.systemd.services.display-manager.enable;
+          message = "No Display Managers must be enabled when jovian.steam.autoStart is used";
+        }
+      ];
+
+      warnings = lib.optional (cfg.desktopSession == null) ''
+        jovian.steam.desktopSession is unset and using the Switch to Desktop function
+        in Gaming Mode will relaunch Gaming Mode.
```

Let's make the warning but as simple as possible, then explain the consequences after.

(We already have a hint after about how to solve, which means I don't have to suggest one!)

```suggestion
        jovian.steam.desktopSession is unset.
        
        This means that using the Switch to Desktop function in Gaming Mode will
        relaunch Gaming Mode.
```